bucket/fun.json

```
@@ -0,0 +1,14 @@
+{
+    "version": "3.0.8",
+    "description": "Complete set of commands to define, develop, test serverless applications, and deploy them to the Alibaba Cloud.",
+    "homepage": "https://github.com/alibaba/funcraft",
```

Lets assume that ID of page will not change

```suggestion
    "homepage": "https://alibabacloud.com/help/doc-detail/64204.html?lang=en",
```